CHANGELOG.rst

```diff
@@ -24,12 +24,6 @@ This document describes changes between each past release.
 - Add the ``protocol_version`` to tell which protocol version is
   implemented by the service. (#324)
 
-**Breaking changes**
-
-- For PostgreSQL backends, it is recommended to specify ``postgresql://``
-
```

Comment:
I just merged the two New Features paragraph, but I can put Breaking change before new features if you'd like to.